ynput/ayon-applications
# Chore: add missing motionbuilder icon

Add missing motionbuilder icon

### Testing Notes
1. Build the application addon with this branch
2. Make sure you include motionbuilder as part of the applications for your project
3. Go to AYON launcher
4. Moition Builder icon should be shown as expected
